NorrisFacts/Support/AppDelegate.swift

```diff
@@ -8,6 +8,13 @@
 
 import UIKit
 
+// Adding some
+// fairly
+// unnnecessary
```

Comment:
What about multiline comments?
```suggestion
/** Adding some
     fairly
     unnnecessary */
```